htdocs/xoops_lib/Xoops/Core/Assets.php

```diff
@@ -333,6 +333,16 @@ public function setDebug($debug)
     }
 
     /**
+     * getDebug enable debug mode, will skip filters prefixed with '?'
```

Comment:
The debug property is private for a reason. It is NOT a getter/setter situation, as the implementation can change. There is no valid reason to expose this detail.